biahub/concatenate.py

```diff
@@ -238,22 +238,32 @@ def calculate_cropped_size(
 def concatenate(
     settings: ConcatenateSettings,
     output_dirpath: Path,
-    sbatch_filepath: str = None,
+    sbatch_filepath: str | None = None,
     local: bool = False,
+    block: bool = False,
```

Comment:
What was the motivation for including the block parameter? Was it useful during testing?

Reply:
When running locally there isn't a good way to check if the jobs (processes) have finished. It is also useful for testing.